README.md

```diff
@@ -61,6 +68,9 @@ Please see the Examples section for more details.
 
 ## Versions
 
+### Unreleased
+* Decoupled VM generation from underlying VHD formar in xVMHyperV resourse.
```

Comment:
Could you please add some more info about the fact it's a breaking change?
Also, I believe you meant "format"